Inspicio/Views/Images/Index.cshtml

```diff
@@ -23,13 +23,13 @@
             <th>
                 @Html.DisplayNameFor(model => model.Description)
             </th>
-
             <th>
-                @Html.DisplayNameFor(model => model.DownRating)
```

Comment:
Consider renaming DownRating/UpRating as NoOfDislikes/NoOfLikes respectively. My understanding is that this displays the total number of likes for an image, so might be better to choose a more descriptive name.